cypress/e2e/Orange/04-orangeHR_Side_Menu_TCs.cy.js

```diff
@@ -0,0 +1,60 @@
+import SideMenu from "../../support/pageObjects/sideMenu";
+Cypress.on('uncaught:exception', () => {
+  return false;
+});
+
+const sidemenu = new SideMenu();
+
+describe('Side Menu', () => {
+
+  beforeEach(function(){
+    cy.login();
+    cy.url().should('eq', Cypress.env('dashboardurl'));
+  });
+
+  it('Collide Side Menu', ()=>{
+    sidemenu.collideButton().click();
+    sidemenu.sideSearchBar().should('not.be.visible');
+  });
+
+  it('Expand side menu', ()=>{
+    sidemenu.collideButton().click();
+    sidemenu.sideSearchBar().should('not.be.visible');
+    sidemenu.collideButton().click();
+    sidemenu.sideSearchBar().should('be.visible');
+  });
+
+  it('Click on the on OrangeHRM icon', ()=>{
+    sidemenu.getBanner().click();
+    const newUrl = cy.url();
+    expect(Cypress.env('baseUrl')).to.not.equal(newUrl);
+  });
+
+  it('Visit all side menu pages', ()=>{
```

Comment:
Since we are not verifying any UI component, instead of going to every page, we could just verify the button link and it should improve running time.

Reply:
@edgarysabel jjsta dded new changes based pn your sugestion